modules.d/90dmsquash-live/dmsquash-live-root.sh

```diff
@@ -124,7 +124,7 @@ else
         # at the first glance, but ends with lots and lots of squashfs
         # errors, because systemd attempts to kill the ntfs-3g process?!
         if [ -x "/usr/bin/ntfs-3g" ]; then
-            (exec -a @ntfs-3g ntfs-3g -o "${liverw:-ro}" "$livedev" /run/initramfs/live) | vwarn
+            ntfs-3g -o "${liverw:-ro}" "$livedev" /run/initramfs/live | vwarn
```

Comment:
Bash's `exec -a @ntfs-3g ...` is intentional:

https://systemd.io/ROOT_STORAGE_DAEMONS/

Reply:
I somehow completely missed the `@`, so it looked superfluous. Considering #1897, would you prefer this to get rolled back to the bash spelling or something like
```sh
(
  ln -s /usr/bin/ntfs-3g /run/@ntfs-3g
  (sleep 1 && rm /run/@ntfs-3g) &
  PATH=/run
  exec @ntfs-3g -o ...
) | vwarn
```

Reply:
\*sigh\* simple feature bash/ksh/zsh's `exec` have. I would avoid that, it is awful.